### ISSUE TITLE
doc/manual.md formatting issue due to missing ```

### ISSUE THREAD

**Issue by JLPLabs** [pallene-lang/pallene#458]
Due to a missing ``` at line 385 the rest of `doc/manual.md` is incorrectly formatted. That is, explanatory text appears as code blocks and code blocks appear as text.

The next code block **should** look like
```
function foo(x : any) : integer
   local y: integer = (x as integer)
   return y + y
end
```

But it actually looks like:

function foo(x : any) : integer local y: integer = (x as integer) return y + y end

----
_Normally I'd submit a pull request, but I've not yet got past reading the documentation, so none of my workflow is setup yet._

**Comment by hugomg** [comment]
Oops! Looks like I messed up the manual last time I edited it.

Would you mind creating a PR for this? I think this might not require setting up a local development environment, because it's only changing the documentation.

By the way, feel free to ask questions if some problem comes up. We recently updated the CONTRIBUTING.md; any feedback is welcome.

**Comment by JLPLabs** [comment]
Please let me know if this PR won't work for you, and if so, what should be fixed. Thanks

**Comment by hugomg** [comment]
Fixed in #459.
